src/js/components/Navigation/Header.jsx

```diff
@@ -584,3 +585,8 @@ Header.propTypes = {
   params: PropTypes.object,
   // pathname: PropTypes.string,
 };
+
+const BackToSettingsMobileDesktopSpan = styled('div')`
```

Comment:
This might want to be a span instead of div.

Reply:
Ok